contracts/MintManager_v2.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity 0.5.16;
+
+import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC721/ERC721.sol";
+import "./interfaces/IERC20Mintable.sol";
+
+contract MintManager_v2 is Initializable {
+    using SafeMath for uint256;
+
+    //mapping(uint256 => IERC20Mintable) public erc20Minter; // set our erc20 token as default
+    // Things we need go here....
+    address public admin;
+    address public steward;
+    IERC20Mintable public token;
+
+    modifier onlySteward() {
+        require(msg.sender == steward, "Not steward");
+        _;
+    }
+
+    modifier onlyAdmin() {
+        require(msg.sender == admin, "Not admin");
+        _;
+    }
+
+    modifier onlyAdminOrSteward() {
```

Comment:
This is currently unused. Maybe comment out because it will clearly be useful in the future.